client/blocks/comment-detail/docs/comment-faker.jsx

```diff
@@ -28,59 +28,101 @@ export const CommentFaker = WrappedCommentList => class extends Component {
 		}
 	}
 
-	deleteCommentPermanently = commentId => this.setState( { comments: omit( this.state.comments, commentId ) } );
+	deleteCommentPermanently = commentId => this.doAction( [ commentId ], { status: 'delete' } );
 
-	getCommentsFromProps = ( { comments } ) => this.setState( { comments: keyBy( comments, 'ID' ) } );
+	/**
+	 * Sets a status and/or a like value to a list of comments.
+	 *
+	 * @param {Array}  commentIds
+	 * @param {Object} action
+	 * @param {bool}   action.i_like
+	 * @param {string} action.status
+	 */
+	doAction = ( commentIds, { i_like, status } ) => {
```

Comment:
Pretty minor since it's in the faker, but this was more about naming in general ;). `doAction`? but which one?